src/components/network/PageNetwork.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <TmPage data-title="Network" class="small" hide-header>
+    <template>
+      <h3>Mainnets</h3>
+      <ul>
+        <li
+          v-for="network in mainNets"
```

Comment:
this 🐫-casing is so fugly

lol

Reply:
Haha. I expanded it to `mainNetworks`, which does look a little nicer.